docs/source/handling_meta.rst

```diff
@@ -31,18 +31,28 @@ Attach meta to a model with the ``setMeta()`` method. The method accepts two arg
 
 ::
 
-	<?php
-	$model->setMeta('key', 'value');
+    <?php
+    $model->setMeta('key', 'value');
 
-To set multiple meta key and value pairs at once, you can pass an associative array or collection to ``syncMeta()``.
+To set multiple meta key and value pairs at once, you can pass an associative array or collection to ``setManyMeta()``. The meta will be added to the model.
 
 ::
 
-	<?php
-	$model->syncMeta([
-		'name' => 'John Doe',
-		'age' => 18,
-	]);
+    <?php
+    $model->setManyMeta([
+        'name' => 'John Doe',
+        'age' => 18,
+    ]);
+
+To replace existing meta with a new set of meta, you can pass an associative array or collection to ``syncMeta()``. **All existing meta will be removed before the new meta is attached.**
```

Comment:
I prefer this wording to your other PR. But I agree the old wording was not clear enough. 👍